src/datastore/pg-store.ts

```diff
@@ -4524,4 +4524,13 @@ export class PgStore extends BasePgStore {
     `;
     if (result.count) return result[0];
   }
+
+  async getStacksBlockCountAtBurnBlock(burnBlockHeight: number): Promise<number> {
+    const result = await this.sql<{ count: number }[]>`
+      SELECT COUNT(*) AS count
+      FROM blocks
+      WHERE burn_block_height = ${burnBlockHeight} AND canonical = TRUE
+    `;
+    return result[0].count ?? 0;
```

Comment:
If an API writer just recently booted up and has not yet received a burn block event, do you know what this metric to show up as? I'm _guessing_ that the prometheus server simply won't display the `stacks_blocks_in_previous_burn_block` metric until a burn block is received and this code path is triggered. Are you able to verify this locally?

I'm asking because I want to avoid the scenario where a writer is restarted and displays the value `0` for a while until it receives the next burn block. This could cause red herrings in our alerts.

Reply:
good catch @CharlieC3 , fixed in latest commit so it first appears as blank instead of `0`